src/discord/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "discord",
+  "version": "0.5.1",
```

Comment:
```suggestion
  "version": "0.1.0",
```
Hi @suekou !
Shouldn't `"version": "0.5.1"` be `version": "0.1.0"` instead?

Reply:
thank you for reviewing!
This is why we need to match the versions. ->https://github.com/modelcontextprotocol/servers/pull/69/files#r1860365172

Reply:
Got it!
thx :)